master/internal/provisioner/aws_config.go

```diff
@@ -34,7 +34,7 @@ type AWSClusterConfig struct {
 	LogGroup  string `json:"log_group"`
 	LogStream string `json:"log_stream"`
 
-	SpotInstanceEnabled bool   `json:"spot_instance_enabled"`
+	SpotInstanceEnabled bool   `json:"spot"`
```

Comment:
blocking: should change this variable name to be `Spot` to match other variables using the camel case version of the json tag to be the names

Reply:
I renamed it to `SpotEnabled`, but I don't think we should call it `Spot` because it isn't descriptive enough for a variable name in code.